server_nginx.html

```diff
@@ -15,36 +15,40 @@
 # Wide-open CORS config for nginx
 #
 location / {
+     # the dance you have to do because if's can't be nested in nginx
+     if ($http_origin) {
+        set $cors "${cors}O"
```

Comment:
you seem right on these . expect a fixed PR soon
On Jun 9, 2015 10:36 PM, "Monsur Hossain" <notifications@github.com> wrote:

> In server_nginx.html
> <https://github.com/monsur/enable-cors.org/pull/103#discussion_r32084103>:
>
> > @@ -15,36 +15,40 @@
> >  # Wide-open CORS config for nginx
> >  #
> >  location / {
> > +     # the dance you have to do because if's can't be nested in nginx
> > +     if ($http_origin) {
> > +        set $cors "${cors}O"
>
> Instead of abbreviations such as 'O', 'P', etc, why not spell things out,
> e.g. OriginOptions, OriginGet, OriginPost. It would make the code more
> readable below.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/monsur/enable-cors.org/pull/103/files#r32084103>.
>